src/rules.mk

```diff
@@ -34,18 +34,20 @@ SPHINX_ARGS     := . $(BUILD_DIR)
 SPHINX_OPTS     := -W -n
 RST2HTML        := $(ENV_DIR)/bin/rst2html.py
 VALE_VERSION    := 2.6.7
-VALE_URL        := https://github.com/errata-ai/vale/releases/download
+ERRATA_AI       := https://github.com/errata-ai
+VALE_URL        := $(ERRATA_AI)/vale/releases/download
 VALE_URL        := $(VALE_URL)/v$(VALE_VERSION)
 VALE_LINUX      := vale_$(VALE_VERSION)_Linux_64-bit.tar.gz
 VALE_MACOS      := vale_$(VALE_VERSION)_macOS_64-bit.tar.gz
 VALE_WIN        := vale_$(VALE_VERSION)_Windows_64-bit.tar.gz
-VALE_PROSELINT  := https://github.com/errata-ai/proselint/archive/v0.3.2.tar.gz
-VALE_WRITEGOOD  := https://github.com/errata-ai/write-good/archive/v0.4.0.tar.gz
+VALE_PROSELINT  := $(ERRATA_AI)/proselint/archive/v0.3.2.tar.gz
+VALE_WRITEGOOD  := $(ERRATA_AI)/write-good/archive/v0.4.0.tar.gz
 NO_VALE_FILE    := $(TOP_DIR)/$(DOCS_DIR)/_no_vale # Vale disabled if exists
 TOOLS_DIR       := $(LOCAL_DIR)/tools
 STYLE_DIR       := $(LOCAL_DIR)/tools/styles
 VALE            := $(TOOLS_DIR)/vale
-VALE_OPTS       := --config=$(SRC_DIR)/_vale.ini --glob='!{**.git/**,**.venv/**,**.crate-docs/**,**vendor/**}' --no-exit
+VALE_OPTS       := --config=$(SRC_DIR)/_vale.ini --no-exit \
+                       --glob='!{**.**/**,**site-packages/,**vendor/**}'
```

Comment:
- `**.**/**` would just exclude everything? How would stuff be **included** again?

Reply:
> While I was at it, I simplified the glob patterns by replacing all directories starting with . with a single glob. There should never be any `.` directory with files we want to lint, and this allows us to reduce the glob specificity, which will in turn improve maintainability.

I see. If that works...

Reply:
yeah, not sure. it might need to be `**/.**/**`?

Reply:
and for that matter, perhaps `**/site-packages/**` and `**/vendor/**` is better? thoughts?

Reply:
We can just try....?

Reply:
Ah, yeah. I believe the `/**` suffix should be there per item.

Reply:
For replacing `**.git/**,**.venv/**`, `**.*/**` will hopefully be okay. Note the single `*` after `.`.

Reply:
> We have to do trial-and-error on that.

I just verified it on `crate-pdo`. That willl be just fine:
```
**/.*/**,site-packages,vendor
```

Reply:
thanks so much for doing trial-and-error! fixed!